components/Giant/Giant.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Text, View, Image } from "react-native";
+import * as Font from "expo-font";
+import styles from "./GiantStyles.js";
+
+export default class Giant extends React.Component {
+  async componentDidMount() {
+    await Font.loadAsync({
+      "amatic-sc": require("../../assets/fonts/amatic-sc.ttf")
+    });
+    this.setState({ assetsLoaded: true });
+  }
+
+  render() {
+    return (
+      <View>
+        <View style={{ flexDirection: "row" }}>
+          <View>
```

Comment:
Is there a point for this View ?

Reply:
Yes. There must be only 1 Component in the render method. This is part of the design of React. That's why we wrap everything in 1 view.